$OpenBSD: patch-pjlib-util_src_pjlib-util-test_resolver_test_c,v 1.1 2010/04/02 11:48:24 dcoppa Exp $

fix from upstream:
fix bug in pjlib-util resolver test on OpenBSD

--- pjlib-util/src/pjlib-util-test/resolver_test.c.orig	Thu Jan  1 21:56:36 2009
+++ pjlib-util/src/pjlib-util-test/resolver_test.c	Wed Mar 31 10:28:45 2010
@@ -813,8 +813,8 @@ static int dns_test(void)
     /* Now only server 0 should get packet, since both servers are
      * in STATE_ACTIVE state
      */
-    pj_assert(g_server[0].pkt_count == 1);
-    pj_assert(g_server[1].pkt_count == 0);
+    pj_assert((g_server[0].pkt_count == 1 && g_server[1].pkt_count == 0) ||
+	      (g_server[1].pkt_count == 1 && g_server[0].pkt_count == 0));
 
     /* Wait to allow probing period to complete */
     PJ_LOG(3,(THIS_FILE, "  waiting for active NS to expire (%d sec)",
